documentation/navmodel/index.md

```diff
@@ -0,0 +1,33 @@
+# Navigation models
+
+Navigation models are one of the most important abstractions of Appyx.
+
+## What's a NavModel?
+
+Navigation models describe navigation itself – by the states and operations they define, any custom navigation mechanism can be implemented. The state is then yours to be controlled from your business logic, which then becomes the state of navigation directly.
```

Comment:
```suggestion
Navigation models describe navigation with two parameters: Routings (Navigation targets) and State (finite states for Navigation targets). You define which operations your Navigation models support and how they change the state of Routings. 
```

Reply:
The original phrasing has some key elements I don't want to discard. We can discuss your additions separately, documentation will need more update.